.github/workflows/unit-tests.yml

```diff
@@ -140,7 +140,7 @@ jobs:
           [[ "${{ matrix.mode }}" = *-deps ]] && mv composer.json.phpunit composer.json || true
 
           if [[ "${{ matrix.mode }}" = low-deps ]]; then
-            echo SYMFONY_PHPUNIT_REQUIRE="nikic/php-parser:^4.18" >> $GITHUB_ENV
+            echo SYMFONY_PHPUNIT_REQUIRE="nikic/php-parser:^5.0" >> $GITHUB_ENV
```

Comment:
Do we still have a job testing our compat with `nikic/php-parser` v4 for our code using it ?

Reply:
I am afraid we cannot: https://github.com/symfony/symfony/pull/58370#discussion_r1835337410:

> PHPUnit 11 (through its dependency on `phpunit/php-code-coverage` is not compatible with `nikic/php-parser` 4

Reply:
Only Translation supports v4, I would bump it to ^5 on 7.3

Reply:
see #60255